x/concentrated-liquidity/types/keys.go

```diff
@@ -126,45 +132,46 @@ func keyTickPrefixByPoolIdPrealloc(poolId uint64, preAllocBytes int) []byte {
 
 // PositionId<>LockId and LockId<>PositionId Prefix Keys
 func PositionIdForLockIdKeys(positionId, lockId uint64) (positionIdToLockIdKey []byte, lockIdToPositionIdKey []byte) {
-	positionIdToLockIdKey = []byte(fmt.Sprintf("%s%s%d", PositionToLockPrefix, KeySeparator, positionId))
-	lockIdToPositionIdKey = []byte(fmt.Sprintf("%s%s%d", LockToPositionPrefix, KeySeparator, lockId))
+	positionIdToLockIdKey = []byte(fmt.Sprintf("%s%d", PositionToLockPrefix, positionId))
+	lockIdToPositionIdKey = []byte(fmt.Sprintf("%s%d", LockToPositionPrefix, lockId))
 	return positionIdToLockIdKey, lockIdToPositionIdKey
 }
 
-// PositionToLockPrefix Prefix Keys
-
+// KeyPositionIdForLock returns the key consisted of (PositionToLockPrefix | position Id)
 func KeyPositionIdForLock(positionId uint64) []byte {
-	return []byte(fmt.Sprintf("%s%s%d", PositionToLockPrefix, KeySeparator, positionId))
+	return []byte(fmt.Sprintf("%s%d", PositionToLockPrefix, positionId))
 }
 
-// LockToPositionPrefix Prefix Keys
-
+// KeyLockIdForPositionId returns the key consisted of (KeyLockIdForPositionId | lockId)
 func KeyLockIdForPositionId(lockId uint64) []byte {
-	return []byte(fmt.Sprintf("%s%s%d", LockToPositionPrefix, KeySeparator, lockId))
+	return []byte(fmt.Sprintf("%s%d", LockToPositionPrefix, lockId))
 }
 
-// PoolIdForLiquidity Prefix Keys
-
-func KeyPoolIdForLiquidity(poolId uint64) []byte {
-	return []byte(fmt.Sprintf("%s%s%d", PoolIdForLiquidityPrefix, KeySeparator, poolId))
+// KeyFullRangeLiquidityPrefix returns the prefix used to keep track of full range liquidity for each pool.
+func KeyFullRangeLiquidityPrefix(poolId uint64) []byte {
+	return []byte(fmt.Sprintf("%s%d", FullRangeLiquidityPrefix, poolId))
 }
 
-// PositionId Prefix Keys
-
+// KeyPositionId returns the prefix the key consisted of (PositionIdPrefix | position Id) and is used to store position info.
 func KeyPositionId(positionId uint64) []byte {
-	return []byte(fmt.Sprintf("%s%s%d", PositionIdPrefix, KeySeparator, positionId))
+	return []byte(fmt.Sprintf("%s%d", PositionIdPrefix, positionId))
```

Comment:
Minor question: doesn't removing these separators make it difficult to parse the key later? e.g. if we wanted to get all positions for a query? Or would the intention be to simply search using the prefix

Reply:
the latter case is the correct case!

Reply:
I see, thanks for clarifying!